src/layer-manager/test/generators/heatmap.test.js

```diff
@@ -0,0 +1,88 @@
+import LayerManagerLib from '../../lib/index'
+import { validate as mapboxStyleValidator } from '@mapbox/mapbox-gl-style-spec'
+import HeatmapGenerator, {
+  HEATMAP_TYPE,
+  GEOM_TYPES,
+  COLOR_RAMPS,
+} from '../../lib/generators/heatmap/heatmap.js'
+
+const FAST_TILES_API = 'https://fst-tiles-jzzp2ui3wq-uc.a.run.app/v1/'
+
+test('returns a valid style for a simple static gridded heatmap', async () => {
```

Comment:
this test was failing:
![image](https://user-images.githubusercontent.com/10500650/72832455-2fe3da00-3c85-11ea-832f-17e09abc1ad5.png)

and I think it worth it to include a prepublish script to test the library before releasing to npm, what are your thoughts ?

Reply:
Good idea and thanks for catching that.